integration-tests/src/test/java/tachyon/client/BufferedBlockInStreamIntegrationTest.java

```diff
@@ -46,19 +46,19 @@
   @ClassRule
   public static LocalTachyonClusterResource sLocalTachyonClusterResource =
       new LocalTachyonClusterResource(Constants.GB, Constants.KB, Constants.GB);
-  private static TachyonFileSystem sTfs;
+  private static FileSystem sTfs;
```

Comment:
Do you mind if I leave renaming for another PR? There will be a large number of renames to do.